demisto_sdk/commands/common/tools.py

```diff
@@ -109,6 +109,9 @@
 colorama.init()  # initialize color palette
 
 
+FILE_SUFFIX_TO_EXTRACT = ["yml", "py", "json"]
```

Comment:
constant name is unclear, please elaborate - why only these two? where is it used?

Reply:
then `GRAPH_SUPPORTED_FILE_TYPES` or `GRAPH_SUPPORTED_CONTENT_FILE_TYPES` would be clearer imo